integrations/maya/build.gradle

```diff
@@ -73,6 +73,9 @@ dependencies {
     implementation "com.google.android.material:material:$materialVersion"
     implementation "androidx.constraintlayout:constraintlayout:$constrainLayoutVersion"
     implementation "androidx.swiperefreshlayout:swiperefreshlayout:$swipeRefreshLayoutVersion"
+    implementation "io.coil-kt:coil:$coilVersion"
+    implementation "androidx.browser:browser:$browserVersion"
+    implementation 'me.grantland:autofittextview:0.2.1'
```

Comment:
`AutofitTextView` seems to be commented out, maybe we don't need this import

Reply:
Thanks, this will be removed.